packages/manager/src/features/CloudPulse/Alerts/AlertsListing/AlertListTable.tsx

```diff
@@ -27,6 +29,10 @@ export interface AlertsListTableProps {
    * The list of alerts to display
    */
   alerts: Alert[];
+  /**
+   * The current state of the alerts grouped by tag
+   */
+  alertsGroupedByTag: boolean;
```

Comment:
```suggestion
  alertsGroupedByTag?: boolean;
```

why can't we make this optional property?

Reply:
what will we achieve by making this optional?

Reply:
@ankita-akamai , we don't need pass this property unnecessarily....if it is required we need to pass either true or false